src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  const accountanting = [];
+
+  const getSum = (x) => {
+    return function(y) {
+      while (true) {
+        if (accountanting.length <= 2 || accountanting.length % 2 === 0) {
+          accountanting.push(x + y);
+          break;
+        } else {
+          accountanting.push(`Bzzz... Error!`);
+          break;
+        }
```

Comment:
Now you can check just `accountanting` value, instead of manipulating array

Leave only one `if`, where you will check condition to return string with message